website/docs/reference/warehouse-setups/vertica-setup.md

```diff
@@ -2,22 +2,29 @@
 title: "Vertica setup"
 id: "vertica-setup"
 meta:
-  maintained_by: Community
-  authors: Matthew Carter, Andy Regan, Andrew Hedengren
-  github_repo: 'mpcarter/dbt-vertica'
+  maintained_by: Vertica
+  authors: Vertica (Former authors:- Matthew Carter, Andy Regan, Andrew Hedengren)
+  github_repo: 'https://github.com/vertica/dbt-vertica/'
   pypi_package: 'dbt-vertica'
-  min_core_version: 'v0.21.0'
+  min_core_version: 'v1.3.0'
   cloud_support: Not Supported
-  min_supported_version: 'Vertica 10.0'
-  slack_channel_name: 'n/a'
-  slack_channel_link: 'https://www.getdbt.com/community/'
+  min_supported_version: 'Vertica v12.0'
```

Comment:
the intent of this field is to say:
> to use this adapter, you must have at least this version of Vertica.
so this change tells me that you used to be able to use Vertica 10.0, but cannot anymore. Is that the case?
```suggestion
  min_supported_version: 'Vertica 10.0'
```